src/components/migrationToV3.stories.mdx

```diff
@@ -437,6 +442,140 @@ When you implement your own theme, you need to use `globalStyle` instead of `glo
 
 ## Device
 
+### Renaming
+
+Below table summarizes all renaming of device management related stuff. Simply use new naming to replace old naming for migration.
```

Comment:
I think the second line can be dropped. Also, for the first, suggest using below statement:
To migrate, use new names summarized in table below for device management APIs and components.

Reply:
Updated